mod-configuration-server/src/test/java/org/folio/rest/RestVerticleTest.java

```diff
@@ -144,6 +138,56 @@ public void beforeEach()
       .get(5, TimeUnit.SECONDS);
   }
 
+
+  @Test
+  public void verifySampleDataLoaded(TestContext testContext) {
+    final Async async = testContext.async();
+    okapiHttpClient.get("http://localhost:" + port + "/configurations/entries?query=module==SETTINGS")
+  .thenAccept(response -> {
+    try {
+      testContext.assertEquals(200, response.getStatusCode(),
+        String.format("Unexpected status code: '%s': '%s'", response.getStatusCode(),
```

Comment:
This formatted string value repeats in many places. You could move it to a constant string and just reuse it.

Reply:
done